pandas/util/_test_decorators.py

```diff
@@ -195,7 +195,9 @@ def skip_if_no(package: str, min_version: Optional[str] = None) -> Callable:
 )
 
 
-def skip_if_np_lt(ver_str, reason=None, *args, **kwds):
+def skip_if_np_lt(
```

Comment:
these could actually be a pytest.mark.skipif type (not sure what that actually is though)

Reply:
but can address that later  if so desired